### PR TITLE
Revert "Refactor OrganizationAP, OrganizationService and Organization…

### PR DESCRIPTION
**Pull request by Jabbar-Hussain**
… model."

This reverts commit 796927dd0c22ab6af75d3c3fc3802218c50ffa13.

### CODE REVIEW

**Review by najlaalhaj — APPROVED** [review]

File: src/main/java/io/orten/nano/impl/API.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package io.orten.nano.impl;
+
+import io.orten.nano.model.Organization;
+import io.orten.nano.util.Database;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+
+
```
Comment (munHunger) [inline src/main/java/io/orten/nano/impl/API.java]: remove duplicate empty line

File: src/main/java/io/orten/nano/impl/API.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+public class API {
+
+
```
Comment (munHunger) [inline src/main/java/io/orten/nano/impl/API.java]: remove duplicate empty line

File: src/main/java/io/orten/nano/model/Organization.java [hunk 3]
```diff
@@ -1,37 +1,38 @@
...
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import org.hibernate.annotations.DynamicUpdate;
+import org.hibernate.annotations.SelectBeforeUpdate;
+
+import javax.persistence.*;
+import java.io.Serializable;
 /*
   represents an organization raising fund for one or more projects
```
Comment (munHunger) [inline src/main/java/io/orten/nano/model/Organization.java]: change to a javadoc syntax, i.e.
/**
   \* represents an organzation...
   */

File: src/main/java/io/orten/nano/model/Organization.java [hunk 4]
```diff
@@ -1,37 +1,38 @@
...
-    @Column(name="contactPersonEmail")
+    public String contactPersonName;
+    @Column(name="contactEmail")
     public String contactPersonEmail;
     @Column(name="password")
     public String password;
     @Column(name="billingInformation")
     public String billingInformation;
-    @Column(name="accountNumber")
+    @Column(name="account_Number")
```
Comment (munHunger) [inline src/main/java/io/orten/nano/model/Organization.java]: Keep the naming convention uniform.
It is normal to keep all columns lowercase and a underscore between each word.
But most importantly: be consistent!

File: src/main/java/io/orten/nano/model/Organization.java [hunk 5]
```diff
@@ -45,20 +46,19 @@ public Organization()
...
-        this.contactPerson = contactPerson;
+        this.contactPersonName = contactPersonName;
         this.contactPersonEmail = contactPersonEmail;
         this.password = password;
         this.billingInformation = billingInformation;
         this.accountNumber = accountNumber;
         this.description = description;
     }
 
+
```
Comment (munHunger) [inline src/main/java/io/orten/nano/model/Organization.java]: Remove empty line

File: src/main/java/io/orten/nano/model/Organization.java [hunk 6]
```diff
@@ -45,20 +46,19 @@ public Organization()
 
     //Bussiness Constructor
```
Comment (munHunger) [inline src/main/java/io/orten/nano/model/Organization.java]: Use javadoc comment style. (se previous comment)

File: src/test/java/io/orten/nano/OrganizationTest.java [hunk 7]
```diff
@@ -38,7 +38,8 @@ public void before(){
 public void saveorganaization()
 
     {
-        Organization org= new Organization("67","microsoft", "NY", "nwerme", "mdf@gmail.com", "newpassword45", "brooklyn",
+        Organization org= new Organization("67","microsoft", "NY", "nwerme","0989769865"
+                , "mdf@gmail.com", "newpassword45", "brooklyn",
                 "154678", "very successful organisation");
         s.save(org);
```
Comment (munHunger) [inline src/test/java/io/orten/nano/OrganizationTest.java:40]: This test is not testing anything.
How do you know that org has actually been saved?
No manual intervention should be needed

File: src/test/java/io/orten/nano/OrganizationTest.java [hunk 8]
```diff
@@ -38,7 +38,8 @@ public void before(){
 public void saveorganaization()
```
Comment (munHunger) [inline src/test/java/io/orten/nano/OrganizationTest.java]: Why is this not indented?

**Review by munHunger — CHANGES_REQUESTED** [review]

File: src/main/java/io/orten/nano/impl/API.java (hunk 1 above)
Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/impl/API.java]: This API file is extra so it is removed now

File: src/main/java/io/orten/nano/impl/API.java (hunk 2 above)
Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/impl/API.java]: this file is also extra so it is also removed

File: src/main/java/io/orten/nano/model/Organization.java (hunk 3 above)
Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/model/Organization.java]: added the comment style as suggested

Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/model/Organization.java]: done

File: src/main/java/io/orten/nano/model/Organization.java (hunk 4 above)
Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/model/Organization.java]: Change to "accountNumber" so it be consistent to over all class Organization

File: src/main/java/io/orten/nano/model/Organization.java (hunk 6 above)
Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/model/Organization.java]: comment style changed

File: src/main/java/io/orten/nano/model/Organization.java (hunk 5 above)
Comment (Jabbar-Hussain) [reply src/main/java/io/orten/nano/model/Organization.java]: removed empty lines

File: src/test/java/io/orten/nano/OrganizationTest.java (hunk 8 above)
Comment (Jabbar-Hussain) [reply src/test/java/io/orten/nano/OrganizationTest.java]: indentation is corrected

File: src/test/java/io/orten/nano/OrganizationTest.java (hunk 7 above)
Comment (Jabbar-Hussain) [reply src/test/java/io/orten/nano/OrganizationTest.java:40]: Leave as it for later refactoring

**Review by munHunger — APPROVED** [review]

**Pull request merged**